diagnostic.md

```diff
@@ -32,23 +35,30 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+getters:
+.text()
+.html()
 
+setters:
+.text('foo')
+.html(<foo>bar</foo>)
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is the active representation of a website. The current state of a website and the ways to manipulate it.
```

Comment:
0.5/1. Your answer is a little vague, although not technically incorrect. This is a better answer: "The DOM is a recursive tree, represented in the browser by the `document` object."